Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -101,3 +109,24 @@
     line height remains unchanged. */
   content: "\200b";
 }
+
+.bocEnumValue input[type=radio] + label
+{
+  margin-bottom: var(--label-alignment-margin);
+}
+
+.bocEnumValue input[type=radio] + label::before
+{
+  margin-bottom: var(--negative-icon-alignment-margin);
+  margin-right: var(--icon-alignment-margin);
+}
+
+.bocEnumValue input[type=radio]:disabled + label
+{
+  margin-bottom: var(--negative-disabled-label-alignment-margin);
+}
+
+.bocEnumValue.radioButtonList.disabled
+{
+  margin-top: var(--negative-icon-alignment-margin);
+}
```

Comment:
You're using a different way to align the radiobutton items compared to the checkboxes. Is this actually needed or can we use the same logic here (actually, also then over in common.css) as we do with checkbox?

Reply:
No, this alignment adjustments are only needed in BocEnumValue.css, therefore it can stay there.